test/Jobs/PauseTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Resque\Plugins\Jobs\Tests;
+
+use Credis_Client;
+use Resque;
+use Resque\Plugins\Jobs\Pause;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * Job tests.
+ *
+ * @package     Resque/Tests
+ * @author      Wedy Chainy <askwedi@wedipedia.org>
+ * @license     http://www.opensource.org/licenses/mit-license.php
+ */
+class JobTest extends PHPUnit_Framework_TestCase
+{
+    protected $resque;
+    protected $redis;
+
+    public function setUp()
+    {
+        $config = file_get_contents(REDIS_CONF);
```

Comment:
I assume it's because that's how the php-resque tests work but I'm all for mocking it